client/src/components/UserSidebar/index.jsx

```diff
@@ -27,7 +27,7 @@ const UserSidebar = () => {
     // ************************************************************************************************************
     // FAKE CONTEXT VARIABLE FOR TESTING PURPOSES - set to true to view user info, set to false to view login forms
     // ************************************************************************************************************
-    let context = {
+    let contexxt = {
```

Comment:
best practice would be to call this something like `testContext` or something to that affect